app/src/org/commcare/activities/UpdateActivity.java

```diff
@@ -39,6 +40,8 @@
     private UpdateTask updateTask;
     private UpdateUIController uiController;
 
+    private boolean proceedAutomatically;
```

Comment:
Does this need to be stored in the instance bundle in case you rotate the screen? Or is it somehow magically stored in the intent?